### ISSUE TITLE
demo app application.properties id setting

### ISSUE THREAD

**Issue by weihongchua** [Nexmo/comms-router#89]
In demo app application.properties, how do one find the id information to be set in the properties file?
is routerId = Ref Id?

comms.routerId=
comms.planId=
comms.queueId=

**Comment by ikrustev** [comment]
Exactly. I admit the name is a bit misleading. Will clarify in the doc and change it in subsequent release.

**Comment by weihongchua** [comment]
@ikrustev  Could you provide some example here. I am trying to get the demo app working.

**Comment by ikrustev** [comment]
@weihongchua here you are. Let me know if you need anything else:

```
# the demo application callback as visible by the router
app.callbackBaseUrl=http://localhost:8080/demo-application/api

# the demo application callback as visible by nexmo (external URL)
app.nexmoCallbackBaseUrl=https://api.comms-router.com/demo-application/api

app.phone=<some phone number>
comms.queueId=queue-ivr
comms.planId=simple-menu
comms.routerUrl=http://localhost:8080/comms-router-web/api
comms.routerId=router-ivr
nexmo.appId=<nexmo api id>
nexmo.appPrivateKey=private.key
app.musicOnHoldUrl=<some music on hold url>
```

**Comment by weihongchua** [comment]
Is the following the answer_url that needs to be configured in Nexmo voice application?

# the demo application callback as visible by nexmo (external URL)
app.nexmoCallbackBaseUrl=_**https://api.comms-router.com/demo-application/api**_

**Comment by ikrustev** [comment]
@angel-popov could you, please, verify the config I've posted here and answer the questions?

**Comment by angel-popov** [comment]
callback parameter is specified in the property. Note that it accepts user and password if any.
app.nexmoCallbackBaseUrl=https://user:password@api.comms-router.com/demo-application/api

In nexmo voice setup - we should add     
https://user:pass@api.comms-router.com/demo-application/api/event    - as event url and
https://user:pass@api.comms-router.com/demo-application/api/answer_inbound - as answer url

When router assigns a task to agent, the router calls task's callback-url. In the body of the request there is info about task, the queue and router.
